app/builders/text_field.rb

```diff
@@ -25,7 +26,11 @@ def labelled_text_field
   end
 
   def value
-    f.text_field attribute, input_options
+    if @multiline
+      f.text_area attribute, input_options.merge({rows: @multiline})
+    else
+      f.text_field attribute, input_options
+    end
```

Comment:
This could be a text_area class the extends text_field.
That way the form can expose a f.text_area_input helper method that is more descriptive than passing in a :multiline param to a text_area helper that then outputs a text area